### PR TITLE
dashboard-tasks

### PR DESCRIPTION
**Pull request by rperez89**

### CODE REVIEW

File: src/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 import React from 'react'
-import { Button, GU, Header, SidePanel, Split } from '@aragon/ui'
+import { Button, GU, Header, SidePanel } from '@aragon/ui'
 
 import BalanceModule from './BalanceModule'
-import DashboardStats from './DashboardStats'
-import TaskTable from './TaskTable'
-import { tasks } from '../../mock-data'
+import Tasks from '../../components/Tasks/Tasks'
+// import { tasks } from '../../mock-data'
```
Comment (fabriziovigevani) [inline src/components/Dashboard/Dashboard.js]:
```suggestion

```

File: src/components/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -57,10 +56,7 @@ function Dashboard() {
         <Welcome />
       )}
 
-      <Split
-        primary={<TaskTable tasks={tasks} />}
-        secondary={<DashboardStats />}
-      />
+      <Tasks fromDashboard />
```
Comment (fabriziovigevani) [inline src/components/Dashboard/Dashboard.js]: What do you think of naming this `noHeader` or something like that so it's more general?

File: src/components/Tasks/Tasks.js [hunk 3]
```diff
@@ -7,10 +7,16 @@ import NoTasks from './NoTasks'
 import { useConnectedAccount } from '../../providers/Web3'
 import useFilteredTasks from '../../hooks/useFilteredTasks'
 
-const Tasks = React.memo(() => {
+const Tasks = React.memo(({ fromDashboard }) => {
   const connectedAccount = useConnectedAccount()
+  console.log('connected accccccount ', connectedAccount)
```
Comment (fabriziovigevani) [inline src/components/Tasks/Tasks.js]:
```suggestion
```

File: src/components/Dashboard/Dashboard.js (hunk 2 above)
Comment (rperez89) [reply src/components/Dashboard/Dashboard.js]: yeah i changed that in a PR against develop in the task-list branch, the name that i used is `onlyTable`

**Pull request merged**